docs/index.md

```diff
@@ -3,12 +3,146 @@
 Atlana is our web interface used to configure and run workflows for processing drone captured data.
 
 ## What's available
-Aside from providing a convienient way for running workflows, we also provide:
-- configure one or more storage locations at the folder and file levels
-- download and restore configured storage locations
-- run workflows
-- create new workflows that use git repositories
-- re-run jobs you've created and jobs others have created
-- download and restore workflows you've created and workflows you've run
-
-Downloaded storage locations and worflows are in JSON format and can be shared with others.
+The UI provides a convienient way for running workflows.
+It also provide ways to:
+
+ * configure one or more storage locations at the folder and file levels
+ * download and restore configured storage locations
+ * run workflows
+ * create new workflows that use git repositories
+ * re-run jobs you've created and jobs others have created
+ * download and restore workflows you've created and workflows you've run
+
+The downloaded configurations of storage locations and workflows are in JSON format and can be shared with others.
+
+## Tutorial
+Items needed:
+
+- an orthomosaic image to be processed
+- a GeoJSON file containing the plots of interest
+- an optional YAML file containing [experiment information](https://osf.io/xdkcy/wiki/Configuration%20YAML/) [![external link](img/external_link.png)](https://osf.io/xdkcy/wiki/Configuration%20YAML/)
+
+What we will  be doing:
+
+1. [Upload files and configure](#uploading-and-configuring-files) a server-side file storage definition
+2. Running the [Canopy Cover image-based workflow](#running-a-workflow)
+3. [Downloading the results](#downloading-the-results) to the local computer
+4. [Deleting](#delete-workflow-run) the workflow run
+
+Note that the geographic boundaries of the image don't need to match that of the plots.
+Only those plots that intersect the image will be processed.
+
+Additionally, the coordinate system of the image doesn't need to match that of the plots.
+Plot points are converted automatically to the image's geographic coordinate system.
+
+### Sample data
+
+We also provide an [archive of sample data](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+) [![external link](img/external_link.png)](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz) that can be downloaded and used in this Tutorial.
+Be sure to extract the files from this archive before uploading them.
+Uploaded archives are **not** usable by the system.
```

Comment:
```suggestion
```

removed for brevity (hopefully the system will let the user know)

Reply:
I would like to keep the sentence. Users find out when the file they want doesn't exist on the server. I feel it's better to warn them ahead of time. Issue: https://github.com/AgPipeline/issues-and-projects/issues/559